### PR TITLE
New queries for mobile and a11y chapters

### PR DESCRIPTION
**Pull request by obto**
New queries that were used in the mobile and a11y chapters

### CODE REVIEW

File: sql/2019/09_Accessibility/09_01_b.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+try {
+  var $ = JSON.parse(payload);
+  var elements = JSON.parse($._element_count);
+  if (Array.isArray(elements) || typeof elements != 'object') {
+    return false;
+  }
+
+  return (elements.h1 || elements.h2 || elements.h3 || elements.h4 || elements.h5 || elements.h6 || 0) > 0;
+} catch (e) {
+  return [];
```
Comment (rviscomi) [inline sql/2019/09_Accessibility/09_01_b.sql]:
```suggestion
  return false;
```

File: sql/2019/09_Accessibility/09_01_b.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#standardSQL
+# 09_01_b: % of pages having any heading
```
Comment (rviscomi) [inline sql/2019/09_Accessibility/09_01_b.sql]: Not a big deal but the convention I've been using has been more like `09_01b.sql`.

**Comment by rviscomi** [comment]
Just a ping to see if you're able to resolve the comment nits. This is good to merge otherwise.

**Comment by obto** [comment]
Yeah I'll be fixing the issues and rerunning the CLS queries we discussed, tonight.

**Pull request merged**